lib/core/card-view/components/card-view-variablesitem/card-view-variablesitem.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="adf-property-label">{{ property.label | translate }}</div>
+<div class="adf-property-value">
+    {{ 'CORE.CARDVIEW.VARIABLES.ADD' | translate }}
+    <button (click)="add()" mat-icon-button class="card-view__variables__add-btn">
+        <mat-icon>add</mat-icon>
+    </button>
+
+    <div class="card-view__variables" *ngIf="variables && variables.length">
+        <div class="card-view__variables__row">
+            <div class="card-view__variables__col">{{ 'CORE.CARDVIEW.VARIABLES.NAME' | translate }}</div>
```

Comment:
Can we use here KEY instead of NAME? And at all the other places?
VARIABLES => KEYVALUEPAIRS